Formula/openj9.rb

```diff
@@ -72,7 +72,7 @@ class Openj9 < Formula
 
   resource "omr" do
     url "https://github.com/eclipse-openj9/openj9-omr.git",
-        tag:      "openj9-0.30.1",
+        tag:      "openj9-0.32.0",
```

Comment:
I don't get how it can update tag while not touching the revision?!

Reply:
Bump doesn't support auto-updating resources; however, the find-and-replace behavior will modify any matching version pattern. Will need to manually update this.